actionpack/lib/action_dispatch/caching.rb

```diff
@@ -0,0 +1,61 @@
+require 'action_dispatch/caching/fragments'
+
+module ActionDispatch
+  module Caching
+    extend ActiveSupport::Concern
+    extend ActiveSupport::Autoload
+
+    module ConfigMethods
+      def cache_store
+        config.cache_store
+      end
+
+      def cache_store=(store)
+        config.cache_store = ActiveSupport::Cache.lookup_store(store)
+      end
+
+      private
+        def cache_configured?
+          perform_caching && cache_store
+        end
+    end
+
+    include AbstractController::Helpers
```

Comment:
Why is this included here? It is being used here somewhere? I think it is fine to remove since both classes that include `AD::Caching` already include this module.